### ISSUE TITLE
No user placeholder translation in authentication popup

### ISSUE THREAD

**Issue by e-mmanuel** [jitsi/jitsi-meet#8219]
### Description:

When authentication enabled and browser with non english language, the "user" placeholder in the popup is not translated.

### Steps to reproduce:

1. Configure Jitsi to enable authentication in room
2. Set-up your browser with non english language
3. Open a room, the popup "Waiting for the host..." appears, click on "I am the host". (obviously the previous texts will appears in the browser language)
4. The placeholder "user" is not translated

### Expected behavior:

The placeholder "user" should be translated

### Actual behavior:

Example in french, "user" is not translated :

![user](https://user-images.githubusercontent.com/18414493/101456128-df747a00-3933-11eb-98cb-c46f8971cadf.png)

### Server information:

- Jitsi Meet version: 2.1.376-g9f12bfe2 (docker version)
- Operating System: Linux

### Client information:

- Browser / app version: Firefox 83, Chrome 87
- Operating System: Linux, Windows

### Additional information:

An entry in lang/main-XX.json is missing in "dialog" section, like "userName"

**Comment by saghul** [comment]
Translations are currently contributed by the community. Any chance you can send a PR?

**Comment by e-mmanuel** [comment]
Hello,

Sure, I can do a PR for lang-XX.json but the front code doesn't read any entry in this files for "user" entry : 
```
<div class="jqimessage aui-dialog2-content">
 <input name="username" type="text" class="input-control" placeholder="user" identity="" autofocus="">
 <input name="password" type="password" class="input-control" data-i18n="[placeholder]dialog.userPassword" placeholder="mot de passe utilisateur">
</div>
```
I guess front code should be modified before any translation ?

**Comment by saghul** [comment]
Yeah, looks that way indeed.

**Comment by damencho** [comment]
Fixed with 70fcabd
